packages/monaco/src/browser/monaco-quick-input-service.ts

```diff
@@ -100,7 +102,7 @@ export class MonacoQuickInputImplementation implements IQuickInputService {
 
     setContextKey(key: string | undefined): void {
         if (key) {
-            this.contextKeyService.createKey<string>(key, undefined);
+            this.contextKey = this.contextKeyService.createKey<string>(key, undefined);
         }
     }
```

Comment:
I believe this method is meant for callers, and it may be problematic for us to set the `contextKey` variable especially since we later rely on it for the `inQuickOpen`.

Reply:
For reference VS Code does not keep a single key stored in this case, but a map of them:
- https://github.com/microsoft/vscode/blob/34d69db22caab3d00c6752bc7a8681239bc0c5d2/src/vs/platform/quickinput/browser/quickInputService.ts#L56
- https://github.com/microsoft/vscode/blob/34d69db22caab3d00c6752bc7a8681239bc0c5d2/src/vs/platform/quickinput/browser/quickInputService.ts#L121-L147